server/controllers/static.ts

```diff
@@ -95,6 +109,48 @@ async function getVideoCaption (req: express.Request, res: express.Response) {
   return res.sendFile(path, { maxAge: STATIC_MAX_AGE })
 }
 
+async function generateNodeinfo (req: express.Request, res: express.Response, next: express.NextFunction) {
+  const { totalVideos } = await VideoModel.getStats()
+  const { totalLocalVideoComments } = await VideoCommentModel.getStats()
+  const { totalUsers } = await UserModel.getStats()
+
+  const json: HttpNodeinfoDiasporaSoftwareNsSchema20 = {
+    version: '2.0',
+    software: {
+      name: 'PeerTube',
```

Comment:
https://github.com/jhass/nodeinfo/blob/master/schemas/2.0/schema.json#L35
The software->name element shouldn't have uppercase chars

Reply:
Thanks! I missed that one.